gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala

```diff
@@ -427,7 +428,8 @@ object ExpressionMappings {
     Sig[CovPopulation](COVAR_POP),
     Sig[CovSample](COVAR_SAMP),
     Sig[Last](LAST),
-    Sig[First](FIRST)
+    Sig[First](FIRST),
+    Sig[HLLVeloxAdapter](APPROX_DISTINCT)
```

Comment:
Add special function signature for each backend is supported. Do we consider moving this to Velox backend only?

Reply:
Moved to velox, please help check if this is right approach.

Reply:
Thank you. LGTM.